hw3/src/routes/List.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import Navbar from "../components/Navbar";
+import axios from 'axios';
+import "../style.css";
+
+//Card containing country info
+const Card = (country) => {
+  //Generic not found response for if the API is missing any data
+  const not_found = "Data not found";
+  console.log(country);
+  return(
+    <div class="card">
+      <h1 class="card-header"> {country['name']} </h1>
+      <img src={country['flag_png']} alt={country['flag_alt'] || "No alternative text provided"} />
```

Comment:
In here you are not destructuring the data correctly, leading to incorrect data handling.

Try destructuring the data like       

> <img src={country.flag_png} alt={country.flag_alt || "No alternative text provided"} />

Reply:
Also try adding

`<img
        crossorigin="anonymous"
        src={country.flag_png}
        alt={country.flag_alt || 'No alternative text provided'}
      />`
      
crossorigin as anonymous, without this you might not be able to access the flag because of CORS